packaging/convert2rhel.spec

```diff
@@ -131,6 +131,12 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
 %attr(0644,root,root) %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Fri Nov 13 2020 Michal Bocek <mbocek@redhat.com> 0.13-1
+- allow conversions of CentOS and Oracle Linux 8
+- fix "TypeError: execve()" py2.6-related error when calling external commands
+- remove unused code related to using offline snapshot or RHEL repositories
+- remove all Red Hat Network (RHN)-related code as RHN has been shut down
+
```

Comment:
Missing these PRs. I'm sure we have more

- https://github.com/oamg/convert2rhel/pull/113
- https://github.com/oamg/convert2rhel/pull/106
- https://github.com/oamg/convert2rhel/pull/109
- https://github.com/oamg/convert2rhel/pull/92
- https://github.com/oamg/convert2rhel/pull/90

Reply:
You're right, I used some trimmed list of merged PRs.

- #113 - this is quite a minor one that I'd say is not worth adding to the changelog
- #106 - I will mention this one
- #109 - this is about fixing a bug that I introduced after the release of 0.12, so it shouldn't appear in the changelog
- #92 - I will mention this one
- #90 - I will mention this one

Some more:
- #83 (Set POSIX/C locale at the start of running the tool) - I'll mention this one
- #117 (Remove python-syspurpose dependency from spec) - I will mention this one
- #84 (Not renaming the original system repofiles anymore) - I will mention this one
- #101 - same as #113 
- #102 - same as #113 
- #97 - same as #113 
- #104 - same as #113 
- #110 - same as #113 
- #111 - same as #113 
- #108 - same as #113
- #103 - same as #113
- #98 - same as #113

One thing I don't understand is why I don't see #84, #109, #103 and #90 as merged in https://github.com/oamg/convert2rhel/commits/master.

Reply:
- #113 - although minor, it improves UX in a minor way. If you want to aggregate UX improvements you can mention it together with #90 and whichever other PR we have that made UX improvements
- #101 - 

#104 #110 #108 - can be aggregated to one change of test infrastructure improvements

> One thing I don't understand is why I don't see #84, #109, #103 and #90 as merged in https://github.com/oamg/convert2rhel/commits/master.

Seems to be because it's co-authored, so it doesn't show PR id in the title/first line
https://github.com/oamg/convert2rhel/commit/e7e66ba72a110fd1ab5ce1b4a6a0e73b2b45b9e9

Reply:
>     #113 - although minor, it improves UX in a minor way. If you want to aggregate UX improvements you can mention it together with #90 and whichever other PR we have that made UX improvements
> 
>     #104 #110 #108 - can be aggregated to one line of test infrastructure improvements

Done.